modules/solace/solace.go

```diff
@@ -0,0 +1,291 @@
+package solace
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/go-connections/nat"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+type SolaceContainer struct {
+	Container      testcontainers.Container
```

Comment:
issue: for consistency across the modules, please use the same pattern, in which the `testcontainers.Container` type is embedded into the module's type.

Reply:
done